static/js/autocomplete.js

```diff
@@ -162,7 +161,7 @@ define(['jquery-nos'], function ($nos) {
                                     // Get the results from an ajax query
                                     else {
                                         post.search = search;
-                                        $this.nosAjax({
+                                        $.ajax({
```

Comment:
nosAjax may add an "error" key that I don't want to be considered as a response in the autocomplete. 
As I don't call the Novius OS API with this ajax call, I don't need to use Novius OS tool to perform this.